src/utils.h

```diff
@@ -51,6 +56,11 @@ typedef enum {I, O, T, Z, S, J, L} BlockType;
 typedef enum {LEFT, NO_ROTATE, RIGHT} Rotation;
 
 /*
+ * Modes that the game can be in
+ */
+typedef enum {RUNNING, PAUSED, SHUTDOWN, BOSS} Gamemode;
```

Comment:
Shouldn't we treat the different types of rotations and modes as constants though?